aws-datastore/src/androidTest/java/com/amplifyframework/datastore/AWSDataStorePluginInstrumentedTest.java

```diff
@@ -181,7 +181,7 @@ public void blogOwnerCreatedAndUpdatedRemotelyIsFoundLocally() throws DataStoreE
 
         // Another HubEvent tells us that an update occurred in the Cloud;
         // the update was applied locally, to an existing record.
-        inboundModelEventAccumulator.take(1);
+        inboundModelEventAccumulator.takeAll();
```

Comment:
Isn't `HubAccumulator` now not-recyclable after this code change? Wouldn't calling `takeAll()` the second time break it?

Reply:
You're right; it is no longer recycle-able. I would expect that calling `takeAll()` the second time would return an empty list immediately.